### PR TITLE
added background  style solid, clear, blur

### PR DESCRIPTION
**Pull request by ziligy**
not sure if you're interested but I added it to my fork

### CODE REVIEW

**Comment by moifort** [comment]
Awesome! It was in my TODO list!

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 Swift Package Manager is integrated within Xcode 11:
 
 1. File → Swift Packages → Add Package Dependency...
-2. Paste the repository URL: https://github.com/moifort/swiftUI-slide-over-card.git
+2. Paste the repository URL: https://github.com/ziligy/swiftUI-slide-over-card.git
```
Comment (moifort) [inline README.md]: Can you put my url :)

File: Sources/SlideOverCard/SlideOverCard.swift [hunk 2]
```diff
@@ -90,12 +96,19 @@ struct Card: ViewModifier {
         
         return ZStack(alignment: .top) {
             ZStack(alignment: .top) {
-                if (colorScheme == .dark) {
-                    Color.black
-                } else {
-                    Color.white
+
+                if backgroundStyle == .blur {
```
Comment (moifort) [inline Sources/SlideOverCard/SlideOverCard.swift:100]: Maybe Switch case is more adapted

Comment (ziligy) [reply Sources/SlideOverCard/SlideOverCard.swift:100]: switch wasn't allowed ??

**Comment by moifort** [comment]
Thank you for your PR!

**Pull request merged**